app/routes/bdb/components/BdbDetail.tsx

```diff
@@ -316,190 +464,54 @@ const BdbDetail = () => {
     >
       <Helmet title={title} />
 
-      <Flex column gap="var(--spacing-md)">
-        <p className={cx(!company.description && 'secondaryFontColor')}>
-          {company.description || 'Ingen beskrivelse tilgjengelig'}
-        </p>
-
-        <div className={styles.infoBubbles}>
-          <InfoBubble
-            icon="briefcase"
-            data={company.companyType}
-            meta="Type bedrift"
-            style={{
-              order: 0,
-            }}
-          />
-          <InfoBubble
-            icon="mail"
-            data={company.paymentMail}
-            meta="Fakturamail"
-            style={{
-              order: 1,
-            }}
-          />
-          <InfoBubble
-            icon="call"
-            data={company.phone}
-            meta="Telefon"
-            style={{
-              order: 2,
-            }}
-          />
-          <InfoBubble
-            icon="at"
-            data={company.website}
-            meta="Nettside"
-            style={{
-              order: 3,
-            }}
-            link={company.website}
-          />
-          <InfoBubble
-            icon="home"
-            data={company.address}
-            meta="Adresse"
-            style={{
-              order: 4,
-            }}
-          />
-          <InfoBubble
-            icon="person"
-            data={(studentContact && studentContact.fullName) || '-'}
-            meta="Studentkontakt"
-            link={
-              studentContact && `abakus.no/users/${studentContact.username}`
-            }
-            style={{
-              order: 5,
-            }}
-          />
-        </div>
+      <ContentSection>
+        <ContentMain className={styles.mainContent}>
+          <CollapsibleDisplayContent
+            content={company.description}
+            skeleton={showSkeleton}
+          ></CollapsibleDisplayContent>
 
-        <div>
-          <h3>
-            Bedriftskontakter{' '}
-            <span className={styles.newestFirst}>(Nyest øverst)</span>
-          </h3>
-          {companyContacts && companyContacts.length > 0 ? (
-            <div className={styles.companyList}>
-              <table className={styles.contactTable}>
-                <thead>
-                  <tr>
-                    <th>Navn</th>
-                    <th>Rolle</th>
-                    <th>E-post</th>
-                    <th>Telefonnummer</th>
-                    <th />
-                  </tr>
-                </thead>
-                <tbody>{companyContacts}</tbody>
-              </table>
-            </div>
+          <Flex justifyContent="space-between" alignItems="center">
```

Comment:
```suggestion
          <Flex alignItems="center">
```
I don't think it's needed. Should some gap be added?

Reply:
In my opinion it looks better to have the button on the right side instead of together with the section title on the left side.

| Right | Left |
| -- | -- |
| ![image](https://github.com/user-attachments/assets/e3e3df7c-1271-4ca8-beec-4fb0f861b551) | ![image](https://github.com/user-attachments/assets/a665e722-aa2f-40bc-99ce-dcb55fb0e5da) |

Reply:
Yes I totally agree! Not sure what I was imagining